### ISSUE TITLE
Mouse hover disappears with keyboard nav hover or vice versa

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molarity#208]
**Test device**
Dell or Bates
**Operating System**
Win 10 or Win 7
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/QA/issues/464. Very minor.
It is possible to see a mouse hover highlight and a keyboard nav highlight in the menu to select a solute. If they over lap and then one moves away then the highlight left behind disappears.

**Steps to reproduce**
1. First, move your mouse to be between the drop down box and the beaker, so that when the menu opens the mouse will be over one of the options.
2. Use keyboard nav to open the menu
3. Use the keyboard nav arrows to move focus to the option highlighted by the mouse.
4. Move either keyboard focus or your mouse to a new selection.

**Visuals**
![highlightgone](https://user-images.githubusercontent.com/41024075/71739699-690ff380-2e17-11ea-822a-a624219da0c3.gif)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Molarity‬
URL: https://phet-dev.colorado.edu/html/molarity/1.5.0-rc.1/phet/molarity_all_phet.html
Version: 1.5.0-rc.1 2019-12-19 01:13:41 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.88 Safari/537.36
Language: en-US
Window: 1536x722
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by zepumph** [comment]
I have seen this before, and I don't think it is a bug. I would want both of those to work individually, so I'm glad they work when using mouse and keyboard at the same time (a rare use case in and of itself). @ariel-phet I would be fine closing this issue, but I want a designer to sign off. What do you think?

**Comment by zepumph** [comment]
Since this also has something to do with behavior of keyboard navigation, tagging @emily-phet for comment too.

**Comment by ariel-phet** [comment]
@KatieWoe @emily-phet @zepumph 

I agree with @zepumph - this is not a bug. Whichever interaction mode you are using takes precedence. In web browsers, this behavior seems to be a well established precedent. The moment you switch to one mode or the other it overides. For instance if you tab navigate github then use your mouse, the keyboard highlight goes away and vice versa.

I think this issue can be safely closed.

**Comment by KatieWoe** [comment]
Quick note though, it only goes away when they overlap. Otherwise you have both at once.

**Comment by zepumph** [comment]
>Otherwise you have both at once.

Right! I see that too. I think after playing with it, I see what you are saying @KatieWoe, you can get an unhighlighted item with focus highlight around it. That said the added complexity that it would take to handle this cross-input-device style is more than I think the issue warrants, especially hearing from @ariel-phet. Closing
